lib/px/utils/pxfilters.lua

```diff
@@ -19,7 +20,7 @@ _M.Whitelist = {};
 -- filter: example.com/api_server_full?data=data
 -- will not filter: example.com/api_server?data=data
 -- _M.Whitelist['uri_full'] = {'/', '/api_server_full' }
-_M.Whitelist['uri_full'] = {}
+_M.Whitelist['uri_full'] = { px_config.custom_block_url }
```

Comment:
Add a comment above that the custom_block_url should not be removed when using the custom block page.